### PR TITLE
add : added quiz page

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
Added quiz page 

image

### CODE REVIEW

File: pages/quiz-page/quiz-page.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+.title {
+    color: var(--pink-primary2);
+    margin-bottom: 2rem;
+}
+
+.ques-score {
+    display: flex;
+    justify-content: space-between;
+    margin-right: 3rem;
+    font-size: 18px;
```
Comment (palash23jain) [inline pages/quiz-page/quiz-page.css]: you might want to use relative unit here (rem/em)

File: pages/quiz-page/quiz-page.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+.options button {
+    font-size: 18px;
+    width: 90%;
+    height: 2rem;
+    border-radius: 5px;
+    margin-bottom: 14px;
+    border: none;
+    cursor: pointer;
+    border: 1px solid var(--grey);
```
Comment (Muzammil-khan710) [inline pages/quiz-page/quiz-page.css]: you are giving border:none as well as border 1px. Fix it ?

File: pages/quiz-page/quiz-page.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+.ques-score {
+    display: flex;
+    justify-content: space-between;
+    margin-right: 3rem;
+    font-size: 18px;
+}
+
+.ques h2 {
+    font-size: 22px;
```
Comment (Muzammil-khan710) [inline pages/quiz-page/quiz-page.css]: can this value converted into rem?

File: pages/quiz-page/quiz-page.css (hunk 1 above)
Comment (Muzammil-khan710) [inline pages/quiz-page/quiz-page.css]: this one too?

**Review by palash23jain — APPROVED** [review]
you might want to remove extra spacing in the code. As of now, this looks good to go

File: pages/quiz-page/quiz-page.css (hunk 2 above)
Comment (JuvitaSaini09) [reply pages/quiz-page/quiz-page.css]: I have made the change, thankyou for review

File: pages/quiz-page/quiz-page.css (hunk 3 above)
Comment (JuvitaSaini09) [reply pages/quiz-page/quiz-page.css]: updated

File: pages/quiz-page/quiz-page.css (hunk 1 above)
Comment (JuvitaSaini09) [reply pages/quiz-page/quiz-page.css]: i have updated it

Comment (JuvitaSaini09) [reply pages/quiz-page/quiz-page.css]: i have updated it,thankyou for the review

**Review by Muzammil-khan710 — APPROVED** [review]
Good to merge! go for it

**Pull request merged**